runtime/primitives/src/currency.rs

```diff
@@ -201,7 +201,10 @@ impl From<u128> for CurrencyId {
 #[cfg(feature = "develop")]
 impl From<CurrencyId> for xcm::latest::Junction {
 	fn from(this: CurrencyId) -> Self {
-		xcm::latest::Junction::GeneralKey(this.encode())
+		xcm::latest::Junction::GeneralKey(sp_runtime::WeakBoundedVec::force_from(
+			this.encode(),
+			None,
+		))
```

Comment:
Why is this changed?

Reply:
This code is enabled by `develop` feature. 
Only udeps uses `--all-features` and catches a compilation error. But the old CI uses `continue-on-error: true` for udeps check.
Perhaps, this code was broken during migration to a new Polkadot version.